src/Template/Adrs/xml/e2b.ctp

```diff
@@ -310,10 +314,10 @@
             </drug>
             <?php  endforeach; ?>
             <summary>
-                <narrativeincludeclinical><?php echo $adr['immediate_cause']; ?></narrativeincludeclinical>
-                <reportercomment><?php echo h($adr['investigations']); ?></reportercomment>
+                <narrativeincludeclinical><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$adr['immediate_cause']); ?></narrativeincludeclinical>
+                <reportercomment><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$adr['investigations']); ?></reportercomment>
                 <senderdiagnosismeddraversion>WHO-ART</senderdiagnosismeddraversion>
-                <senderdiagnosis><?php echo h($adr['diagnosis']); ?></senderdiagnosis>
+                <senderdiagnosis><?php echo preg_replace('/[^A-Za-z0-9\s+]/', '',$adr['diagnosis']); ?></senderdiagnosis>
```

Comment:
Create a utility method that will perform the replace function and reuse it as appropriate